.env

```diff
@@ -1 +1 @@
-REACT_APP_LIFF_ID=1656508316-k7jNojXm
+REACT_APP_LIFF_ID="1656508316-k7jNojXm"
```

Comment:
💭 Originally, it would be better to use a file name such as `.env.example`. However, I think this repository is also used for deploying the actual LIFF Playground, so I've kept this file name.